.npmignore

```diff
@@ -1,7 +1,8 @@
 *
 !dist/**/*
-dist/esm/__tests__/**/*
-dist/cjs/__tests__/**/*
+dist/esm/__tests__
```

Comment:
This file can probably be deleted now.

Reply:
I see you have two `.npmignore` in the root and packages/rooks.
I guess we only need one of them

I checked with
```sh
# in packages/rooks
npm pack --dry-run
```
and removing either one of them seems to work fine.

Reply:
Is that what you meant with
> This file can probably be deleted now.

?

Reply:
@imbhargav5 What is your workflow to publish rooks to npm?
Do you use `npm publish` or let some tools handle it?